src/services/performanceDbApi.js

```diff
@@ -263,6 +263,20 @@ export default {
     AND rle.pipeline = '${dataset}'`
   },
 
+  activeResourcesForOrganisationAndDatasetQuery: (lpa, dataset) => {
+    return /* sql */`
+      select
+        rhe.endpoint, rhe.endpoint_url, rhe.resource, rhe.status
+      from
+        reporting_historic_endpoints rhe
+      LEFT JOIN resource_organisation ro ON rhe.resource = ro.resource
+      LEFT JOIN organisation o ON REPLACE(ro.organisation, '-eng', '') = o.organisation
+      WHERE REPLACE(ro.organisation, '-eng', '') = '${lpa}'
+      AND pipeline = '${dataset}'
+      AND rhe.endpoint_end_date == ''
```

Comment:
```suggestion
      AND (rhe.endpoint_end_date == '' OR rhe.endpoint_end_date is NULL)
```

sometimes it may be null